tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -802,7 +809,7 @@ void test_parse_errors() {
 }
 }  // namespace
 
-// [[Timeout, 15]]
+// [[Timeout, 20]]
```

Comment:
Is it the quaternion ODE integration that slows this down? Is that fine?

Reply:
I saw your explanation above. So the answer is that in an evolution the ODE integration only happens occasionally and hence doesn't slow things down a lot, but in this test it's called an excessive number of times?